chain-tx-enclave-next/enclave-ra/ra-sp-client/src/client.rs

```diff
@@ -65,8 +65,9 @@ impl SpRaClient {
     pub fn get_attestation_report(
         &self,
         quote: Vec<u8>,
+        nonce: [u8; 16],
     ) -> Result<AttestationReport, SpRaClientError> {
-        let request = Request::GetAttestationReport { quote };
+        let request = Request::GetAttestationReport { quote, nonce };
         serde_json::to_writer(&self.stream, &request)?;
 
         let response: Response = serde_json::Deserializer::from_reader(&self.stream)
```

Comment:
should also verify the response contains the same nonce ?

Reply:
Yes, that seems correct according to the docs. Thanks